include/exadg/incompressible_navier_stokes/user_interface/parameters.cpp

```diff
@@ -385,9 +396,32 @@ Parameters::check(dealii::ConditionalOStream const & pcout) const
     }
   }
 
+  if(spatial_discretization == SpatialDiscretization::HDIV)
+  {
+    AssertThrow(
+      use_divergence_penalty == false || use_continuity_penalty == false,
+      dealii::ExcMessage(
+        "The use of penalty terms does not make sense in the case of HDIV since they are evaluated to zero. They should therefore be deactivated."));
+    if(temporal_discretization != TemporalDiscretization::BDFCoupledSolution)
+      AssertThrow(
+        preconditioner_velocity_block == MomentumPreconditioner::None ||
+          preconditioner_velocity_block == MomentumPreconditioner::PointJacobi,
+        dealii::ExcMessage(
+          "Use either PointJacobi or None as preconditioner for the momentum block in the case of HDIV - Raviart-Thomas."));
```

Comment:
I think we need the same check for the viscous step of the dual splitting scheme (i.e. only PointJacobi allowed as preconditioner).